src/main/java/wolox/training/TrainingApplication.java

```diff
@@ -2,6 +2,9 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+
+@EntityScan("org.wolox.training.models.Book")
```

Comment:
This is only necessary if the entity is outside of the java package, meaning, src/main/java.
Given yours is in it, it should work without this. Check it out!